things/tracing/groups.go

```diff
@@ -8,24 +8,15 @@ import (
 )
 
 const (
-	saveGroupOp                    = "save_group"
-	updateGroupOp                  = "update_group"
-	removeGroupOp                  = "remove_group"
-	retrieveAllOp                  = "retrieve_all"
-	retrieveGroupByIDOp            = "retrieve_group_by_id"
-	retrieveGroupByIDsOp           = "retrieve_group_by_ids"
-	retrieveByOwnerOp              = "retrieve_by_owner"
-	retrieveThingMembershipOp      = "retrieve_thing_membership"
-	retrieveChannelMembershipOp    = "retrieve_channel_membership"
-	retrieveGroupThingsOp          = "retrieve_group_things"
-	retrieveGroupThingsByChannelOp = "retrieve_group_things_by_channel"
-	retrieveGroupChannelsOp        = "retrieve_group_channels"
-	assignThingOp                  = "assign_thing"
-	unassignThingOp                = "unassign_thing"
-	assignChannelOp                = "assign_channel"
-	unassignChannelOp              = "unassign_channel"
-	retrieveAllThingRelationsOp    = "retrieve_all_thing_relations"
-	retrieveAllChannelRelationsOp  = "retrieve_all_channel_relations"
+	saveGroupOp             = "save_group"
+	updateGroupOp           = "update_group"
+	removeGroupOp           = "remove_group"
+	retrieveAllOp           = "retrieve_all"
+	retrieveGroupByIDOp     = "retrieve_group_by_id"
+	retrieveGroupByIDsOp    = "retrieve_group_by_ids"
+	retrieveByOwnerOp       = "retrieve_by_owner"
+	retrieveGroupThingsOp   = "retrieve_group_things"
+	retrieveGroupChannelsOp = "retrieve_group_channels"
```

Comment:
Naming is not consistent. For example: `retrieveGroupThingsOp` -> `retrieveThingsByGroupOp`